src/version.js

```diff
@@ -0,0 +1,3 @@
+// Update this when updating package.json
+const sdkVersion = '1.17.0';
```

Comment:
It's not ideal to have to remember to update both versions. Is there any other way to get the version from a single place at runtime? Perhaps via the [environment npm config values](https://docs.npmjs.com/cli/v6/using-npm/scripts#packagejson-vars)?

Reply:
Those don't work runtime, I think. Probably possible to wrap somehow in build script that generate a `js` file, though. I'll leave it for future improvement, since releases aren't super frequent and it's in the instructions anyway.